conf/test_taxonomy.config

```diff
@@ -34,7 +35,10 @@ params {
     run_amp_screening       = true
 
     run_bgc_screening       = true
+    bgc_mincontiglength     = 1000
+    bgc_savefilteredcontigs = true
     bgc_skip_deepbgc        = true
+    bgc_antismash_contigminlength = 1000
```

Comment:
Let's indent everything by 5 spaces to comply with the Harshil alignment :tm: ?

Reply:
Done in [5a31a51](https://github.com/nf-core/funcscan/pull/382/commits/5a31a511466d83862c1b16925e57106104e8018b). No clue why this comment is not shown as outdated :shrug: